rental.rb

```diff
@@ -4,8 +4,8 @@ class Rental
   def initialize(date, person, book)
     @date = date
     @book = book
```

Comment:
Good job working on the project requirements so far. I noticed whenever I try to create a rental you app crushes and throws an error it would be nice if you fixed this issue:

This is the error its displaying:
![Screen Shot 2022-11-04 at 07 28 49](https://user-images.githubusercontent.com/79506105/199897982-3a67dcfb-3cf4-4f78-a86e-db31b41fdda8.png)

Reply:
Hello sir, I think you forgot to put the teacher id you can also check my screen shoot below:
![199897982-3a67dcfb-3cf4-4f78-a86e-db31b41fdda8](https://user-images.githubusercontent.com/45226848/199904935-696ef1f0-a261-428b-b139-55115cf641ab.png)

Reply:
This occurred whenever I tried to add the date.

Reply:
The thing you are pointing at was when I tried to list people and it displayed that